tests/e2e/e2e_ibc_test.go

```diff
@@ -208,6 +208,74 @@ func (s *IntegrationTestSuite) createChannel() {
 	s.T().Logf("IBC transfer channel created between chains %s and %s", s.chainA.id, s.chainB.id)
 }
 
+func (s *IntegrationTestSuite) completeChannelHandshakeFromTry(
```

Comment:
Can we get a docstring why this is needed?

Most code is not commented because it's self explanatory. This function says what it does but not why it's needed. It seems like an ICA detail.

Reply:
Fixed in https://github.com/cosmos/gaia/pull/3001/commits/b7569cc56ae79a31bbf8218973738e3733c0f304